packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,152 @@
+
+Usage guidance for the `error-visible` state will be created, as its usage may be unclear.
+
+### Angular integration
+
+Angular directives will be created for these components. Neither will have form association, so `ControlValueAccessor`s will not be created.
+
+### Blazor integration
+
+Blazor wrappers will be created for both components.
+
+### Additional requirements
+
+-   _User interaction: Do the FAST component's behaviors match the visual design spec? When they differ, which approach is preferable and why?_
+    -   Nesting Accordions: This is not covered in FAST's examples, but in their [spec](https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion/accordion.spec.md) they said that "Ideally this behavior will "just work" and no special behaviors will need to be added to enable/support this. The expectation here should be that an accordion takes content and whatever content is inside the accordion panel respects its own interaction model." This functionality was tested within FAST's example by creating a new accordion inside of one of the accordion items, and it functions. The only exception is when expand-mode="single" is used, which is further discussed in the open issues section. Using nested accordions would be preferred, as this allows more to be done with accordions and the ability to include more information without having to make one long accordion.
+    -   No Additional Requirements
+-   _Styling: Does FAST provide APIs to achieve the styling in the visual design spec?_
+    -   No Additional Requirements
+-   _Testing: Is FAST's coverage sufficient? Should we write any tests beyond Chromatic visual tests?_
+    -   No Additional Requirements
+-   _Documentation: Any requirements besides standard Storybook docs and updating the Example Client App demo?_
+    -   No Additional Requirements
+-   _Tooling: Any new tools, updates to tools, code generation, etc?_
+    -   No Additional Requirements
+-   _Accessibility: keyboard navigation/focus, form input, use with assistive technology, etc._
+    -   These are [aria's guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) for keyboard interactions: enter or space for actions, tab and shift + tab for moving up and down, arrow keys for moving up and down between headers, and home / end (optional) for moving to the first and last accordion header, respectively. I tested all of these in the FAST component explorer, and they're functional. It is expected that these interactions will still be functional when implemented in Nimble.
+-   _Globalization: special RTL handling, swapping of icons/visuals, localization, etc._
+    -   No Additional Requirements
+-   _Performance: does the FAST component meet Nimble's performance requirements?_
+    -   No Additional Requirements
+-   _Security: Any requirements for security?_
+    -   No Additional Requirements
+
+---
+
+## Open Issues
+
```

Comment:
Before approval we should come up with a plan to resolve these open issues. It could be some combination of:
1. file issues to FAST to track them as bugs
2. fix the issues in FAST or find workarounds
3. disable (or just don't document) features like single expand mode or nesting in our first implementation until these behaviors are fixed
4. decide we like some of these behaviors 

Opening this thread for opinions but as I mentioned in standup I'd vote to keep this moving by cutting scope if possible. (We'd still want to file issues to FAST if we did that)

Reply:
We discussed a plan to resolve these issues in today's meeting, and I documented that plan in the spec. 👍